lib/defaults.js

```diff
@@ -29,6 +29,7 @@ const defaults = {
     host: process.env.REDIS_HOST || '127.0.0.1'
   },
   session: {
+    name: 'hof.sid',
```

Comment:
Is this something that should be optionally overridden by the user so that they can name the cookie something relevant to their service?